File: Site/Presentation/TransApp.Site.FrontEnd/src/app/address/models/address-location-model.ts
export class AddressLocationModel {
    public latitude: number;
    public latitudeStr: string;
    public longitude: number;
    public longitudeStr: string;
    public countryCode: string;
    public country: string;
    public city: string;
    public cityCode: string;
    public state: string;
    public stateCode: string;
    public street: string;
    public streetNumber: number;
    public zipCode: string;

    public phone: string;
    public openNow: string;
    public openingHours: string;

    public mapZoom :number = 15;
}

File: Domain/TransApp.Domain/Addresses/AddressLocationModel.cs
namespace TransApp.Domain.Addresses
{
    public class AddressLocationModel
    {
        public decimal Latitude { get; set; }

        public decimal Longitude { get; set; }

        public string CountryCode { get; set; }

        public string Country { get; set; }

        public string City { get; set; }

        public string CityCode { get; set; }

        public string State { get; set; }

        public string StateCode { get; set; }

        public string Street { get; set; }

        public string StreetNumber { get; set; }

        public string ZipCode { get; set; }

    }
}
